### PR TITLE
iosxr_prefix_lists: Add missing keys ge, eq, le

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

File: plugins/modules/iosxr_prefix_lists.py [hunk 1]
```diff
@@ -918,6 +928,53 @@
...
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+commands:
+  description: The set of commands pushed to the remote device.
+  returned: when I(state) is C(merged), C(replaced), C(overridden), C(deleted) or C(purged)
+  type: list
+  sample:
+    - sample command 1
```
Comment (KB-perByte) [inline plugins/modules/iosxr_prefix_lists.py:951]: add relevant commands here

File: plugins/modules/iosxr_prefix_lists.py [hunk 2]
```diff
@@ -88,6 +97,7 @@
     - rendered
     default: merged
 """
+
 EXAMPLES = """
 # Using merged
```
Comment (KB-perByte) [inline plugins/modules/iosxr_prefix_lists.py:102]: can we add an example for the change

**Review by KB-perByte — CHANGES_REQUESTED** [review]

File: plugins/modules/iosxr_prefix_lists.py (hunk 2 above)
Comment (ashwini-mhatre) [reply plugins/modules/iosxr_prefix_lists.py:102]: ack

File: plugins/modules/iosxr_prefix_lists.py (hunk 1 above)
Comment (ashwini-mhatre) [reply plugins/modules/iosxr_prefix_lists.py:951]: ack

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Review by KB-perByte — APPROVED** [review]

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Pull request merged**